mlflow/java/pom.xml

```diff
@@ -49,6 +49,26 @@
       <version>2.22.0</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.sparkjava</groupId>
+      <artifactId>spark-core</artifactId>
+      <version>2.7.2</version>
+    </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
```

Comment:
These  SLF4J packages are included to satisfy `com.sparkjava`s dependency on specific versions of the SLF4J logging library